receiver/kafkareceiver/kafka_receiver_test.go

```diff
@@ -708,82 +708,136 @@ func TestNewMetricsReceiver(t *testing.T) {
 
 func TestComponentStatus(t *testing.T) {
```

Comment:
This test is significantly slower now than on main, we should probably look into that before merging.

Reply:
Looking into it: we're always hitting the deadlines for status events. The problem is that we apparently never implemented component status reporting for the franz-go client implementation. I think we should fix that, and then this test should remain more or less the same regardless of whether franz-go is used.

Reply:
Thanks, I think the changes look good. Can we update this test now, so it doesn't have the featuregate checks?